napari/_qt/_qapp_model/qactions/_view.py

```diff
@@ -66,6 +67,25 @@ def _get_current_activity_dock(window: Window):
         enablement=sys.platform != 'darwin',
         status_tip=trans._('Show/Hide Menubar'),
     ),
+    Action(
+        id=CommandId.TOGGLE_THEME,
+        title=CommandId.TOGGLE_THEME.title,
+        menus=[
+            {
+                'id': MenuId.MENUBAR_VIEW,
+                'group': MenuGroup.NAVIGATION,
+                'order': 1,
+            }
+        ],
+        callback=_viewer_key_bindings.toggle_theme,
+        keybindings=[
+            {
+                'primary': get_settings()
+                .shortcuts.shortcuts['napari:toggle_theme'][0]
```

Comment:
did we have any issues tracking the need to refresh menus after updating shortcuts in settings?

Reply:
I don't see one. Looking at the existing menus, all the keybinds are hard-coded and not from settings—at least as far as I can tell. But especially if we expand some functionality—add a layer menu to expose some of the layer actions?—then we'll need keybinds from settings in menus.
I'll make an issue.